netci.groovy

```diff
@@ -8,6 +8,7 @@ def project = GithubProject
 
 // Map of os -> osGroup.
 def osGroupMap = ['Ubuntu':'Linux',
+                  'Ubuntu15.10':'Linux',
```

Comment:
As noted on the coreclr one, at some point it'd be good to rename the existing ubuntu case.  Not blocking for this PR, though.